Tests/FunctionalTests/fss-search-query.spec.ts

```diff
@@ -11,7 +11,7 @@ describe('Test Search Query Scenario On Search Page', () => {
   let context: BrowserContext;
   let page: Page;  
 
-  beforeEach(async () => {
+  beforeAll(async () => {
```

Comment:
I noticed that we are using `beforeAll` instead of `beforeEach` in the test setup and then needing to `page.reload()` at the beginning of each test. This means that the tests are not independent of each other (potentially causing fragility) and increases the boilerplate required.
I suggest that the browser launch is done in a `beforeAll` in order to decrease the launch time of the tests but that the `browser.newContext()` and other subsequent code is done in a `beforeEach`

Reply:
Updated the script. Thanks